modules/nf-core/multiqc/main.nf

```diff
@@ -0,0 +1,53 @@
+process MULTIQC {
+    label 'process_single'
+
+    conda "bioconda::multiqc=1.13"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/multiqc:1.13--pyhdfd78af_0' :
+        'quay.io/biocontainers/multiqc:1.13--pyhdfd78af_0' }"
```

Comment:
modules/local/multiqc.nf uses 1.12, is is required to have 2 different versions?

Reply:
no, you guys still have a 'custom' MultiQC module - this one isn't used, but I've left it to make template merges easier :)